docs/release-notes/8.14.asciidoc

```diff
@@ -1,6 +1,18 @@
 [[release-notes-header-8.14.0]]
 == 8.14
 
+[discrete]
+[[release-notes-8.14.1]]
+=== 8.14.1
+
+[discrete]
+[[bug-fixes-8.14.1]]
+==== Bug fixes
+
```

Comment:
I think we're missing these 2 PRs:
- fix analyzer preview navigation to timeline https://github.com/elastic/kibana/pull/184150
- fix formatting in event details table https://github.com/elastic/kibana/pull/185899

The rest loogs good!

Reply:
Thanks, @PhilippeOberti! I just both of those in [a6674dd](https://github.com/elastic/security-docs/pull/5372/commits/a6674ddf13af52dbdda687ecdafeda07383da759).